client/state/selectors/get-jetpack-credentials-test-status.js

```diff
@@ -0,0 +1,5 @@
+import 'calypso/state/jetpack/init';
+
+export default function getJetpackCredentialsTestStatus( state, siteId, role ) {
+	return state.jetpack?.credentials?.testRequestStatus?.[ siteId ]?.[ role ] || 'pending';
```

Comment:
Given that we're loading the Jetpack reducer on the first line above, I don't think there's a scenario where `state.jetpack` will not exist, so we should be able to ditch the optional chaining there. Same for the `credentials` subreducer - if the Jetpack reducer is loaded, it always loads the `credentials` subreducer, so that should always be available. Same for the `credentials` subreducer - if the Jetpack reducer is loaded, it always loads the `testRequestStatus` subreducer, so that should always be available.

```suggestion
	return state.jetpack.credentials.testRequestStatus.[ siteId ]?.[ role ] || 'pending';
```

Note: if our tests are failing because those optional chaining instances are missing, then our tests are likely not correctly specifying the empty state scenario - again, in mind with the fact that `state.jetpack` will always be there when this selector is called.

Reply:
Thanks! It is now fixed.